apps/server/src/modules/common-cartridge/common-cartridge-client/card-client/mapper/card-response.mapper.ts

```diff
@@ -0,0 +1,180 @@
+import {
+	CardListResponse,
+	CardResponse,
+	CardResponseElementsInner,
+	VisibilitySettingsResponse,
+	TimestampsResponse,
+	DeletedElementContent,
+	SubmissionContainerElementContent,
+	DrawingElementContent,
+	ExternalToolElementContent,
+	FileElementContent,
+	LinkElementContent,
+	RichTextElementContent,
+} from '../cards-api-client';
+import { CardResponseDto } from '../dto/card-response.dto';
+import { CollaborativeTextEditorElementResponseDto } from '../dto/collaborative-text-editor-element-response.dto';
+import { DeletedElementResponseDto } from '../dto/deleted-element-response.dto';
+import { DrawingElementContentDto } from '../dto/drawing-element-content.dto';
+import { DrawingElementResponseDto } from '../dto/drawing-element-response.dto';
+import { ExternalToolElementContentDto } from '../dto/external-tool-element-content.dto';
+import { ExternalToolElementResponseDto } from '../dto/external-tool-element-response.dto';
+import { FileElementContentDto } from '../dto/file-element-content.dto';
+import { FileElementResponseDto } from '../dto/file-element-response.dto';
+import { LinkElementContentDto } from '../dto/link-element-content.dto';
+import { LinkElementResponseDto } from '../dto/link-element-response.dto';
+import { RichTextElementContentDto } from '../dto/rich-text-element-content.dto';
+import { RichTextElementResponseDto } from '../dto/rich-text-element-response.dto';
+import { SubmissionContainerElementResponseDto } from '../dto/submission-container-element-response.dto';
+import { ContentElementType } from '../enums/content-element-type.enum';
+import { SubmissionContainerElementContentDto } from '../dto/submission-container-element-content.dto';
+import { DeletedElementContentDto } from '../dto/deleted-element-content.dto';
+import { VisibilitySettingsResponseDto } from '../dto/visibility-settings-response.dto';
+import { TimestampResponseDto } from '../dto/timestamp-response.dto';
+import { CardResponseElementsInnerDto } from '../types/card-response-elements-inner.type';
+import { CardListResponseDto } from '../dto/card-list-response.dto';
+
+export class CardResponseMapper {
+	public static mapToCardListResponseDto(cardListResponse: CardListResponse): CardListResponseDto {
+		const cardListResponseDto: CardListResponseDto = new CardListResponseDto([]);
+
+		cardListResponse.data.forEach((cardResponse) => {
+			cardListResponseDto.data.push(this.mapToCardResponseDto(cardResponse));
+		});
+		return cardListResponseDto;
+	}
+
+	private static mapToCardResponseDto(cardResponse: CardResponse): CardResponseDto {
+		return new CardResponseDto(
+			cardResponse.id,
+			cardResponse.title as string,
```

Comment:
this can cause a runtime error if title is undefind (which could be the case). use please the following code instead
```suggestion
			cardResponse.title ?? '',
```

Reply:
DOne